odoo/addons/base/views/res_partner_views.xml

```diff
@@ -81,10 +81,21 @@
             <field name="model">res.partner</field>
             <field name="arch" type="xml">
                 <form string="Contact">
+                    <field name="avatar_128" invisible="1"/>
+                    <field name="city" invisible="1"/>
+                    <field name="comment" invisible="1"/>
```

Comment:
Do we need that much fields :D ? I suppose it comes from CRM allowing to populate fields from the lead ? In which case maybe worth a small comment ?